pi_controller/cattown/src/cattown/cv/vision.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+from ..core.types import CatState
+
+
+class ComputerVisionSystem:
+
+    def initialize_hardware():
+        """
+        Set up the camera.
+        """
+        raise NotImplementedError
+    
+    def get_cat_state(self) -> List[CatState]:
+        """
+        Tell us what the cat(s) are up to. Should return one CatState per identified cat.
+        """
```

Comment:
If we do decide to use CV to Detect mouse captures, we'll need mouse states here as well, too?

Reply:
Yyyyyeeeeeaaaaaaaah. Yes. Yes we will. I haven't really thought that one through yet... started trying to represent it and heard Zoda's voice in my head reminding me to KISS and iterate